project/ProjectPlugin.scala

```diff
@@ -58,7 +58,8 @@ object ProjectPlugin extends AutoPlugin {
       val scalacheck: String         = "1.14.3"
       val scalacheckToolbox: String  = "0.3.1"
       val scalamock: String          = "4.4.0"
-      val scalatest: String          = "3.0.8"
+      val scalatest: String          = "3.1.0"
+      val scalatestplusScheck        = "3.1.0.0-RC2"
```

Comment:
The dependency is not aligned with scalatest? Should we add the type to make it consistent?

Reply:
No, it's not. From 3.1.0 they were separated under its own versioning.